gravitee-apim-rest-api/gravitee-apim-rest-api-service/src/main/java/io/gravitee/rest/api/service/converter/PlanConverter.java

```diff
@@ -21,6 +21,7 @@
 import io.gravitee.definition.model.DefinitionVersion;
 import io.gravitee.definition.model.Rule;
 import io.gravitee.definition.model.flow.Flow;
+import io.gravitee.definition.model.v4.ApiType;
```

Comment:
This import can be removed since it is unused below

Reply:
Done